doc/source/cookbook/complex_plots.rst

```diff
@@ -39,42 +39,42 @@ is inherited by
 controls the number of resolution elements in the
 :class:`~yt.visualization.fixed_resolution.FixedResolutionBuffer`,
 which can be thought of as the number of individually colored
-squares (on a side) in a 2D image.  `buff_size` can be set
+squares (on a side) in a 2D image. ``buff_size`` can be set
 after creating the image with
 :meth:`~yt.visualization.plot_window.PlotWindow.set_buff_size`,
-or during image creation with the `buff_size` argument to any
+or during image creation with the ``buff_size`` argument to any
 of the four preceding classes.
 
-2. `figure_size`, which can be altered with either
+2. ``figure_size``, which can be altered with either
 :meth:`~yt.visualization.plot_container.PlotContainer.set_figure_size`
 or with :meth:`~yt.visualization.plot_container.PlotWindow.set_window_size`
 (the latter simply calls
 :meth:`~yt.visualization.plot_container.PlotContainer.set_figure_size`),
-or can be set during image creation with the `window_size` argument.
+or can be set during image creation with the ``window_size`` argument.
 This sets the size of the final image (including the visualization and,
 if applicable, the axes and colorbar as well) in inches.
 
-3. `dpi`, i.e. the dots-per-inch in your final file, which can also
+3. ``dpi``, i.e. the dots-per-inch in your final file, which can also
 be thought of as the actual resolution of your image.  This can
-only be set on save via the `mpl_kwargs` parameter to
+only be set on save via the ``mpl_kwargs`` parameter to
 :meth:`~yt.visualization.plot_container.PlotContainer.save`.  The
-`dpi` and `figure_size` together set the true resolution of your
-image (final image will be `dpi` :math:`*` `figure_size` pixels on a
-side), so if these are set too low, then your `buff_size` will not
+``dpi`` and ``figure_size`` together set the true resolution of your
+image (final image will be ``dpi`ù :math:`*` ``figure_size`` pixels on a
```

Comment:
Not sure why the prehook stuck a ``ù`` here, but I think it just needs to swap out to be a `.

```suggestion
image (final image will be ``dpi`` :math:`*` ``figure_size`` pixels on a
```

Reply:
The hook merely reported stuff, then I used my clumsy human hands to fix it. The err is human :)
Thanks for catching it !

Reply:
Fun fact: French keyboards have a `ù` alt key, though there's exactly one word in French that uses it: "où" (where).